### DISCUSSION TITLE
How do I keep this browser updated?

### DISCUSSION THREAD

**Discussion by mfdemicco** [Alex313031/Mercury#182] [Q&A]
There doesn't seem to be a link to a repository that I can put into my apt sources file.

**Comment by rafalkrupinski** [comment]
Any chance to put it on build.opensuse.org? It seems to be working well with APT

**Reply by AnxiousPleaseHelp** [reply]
> Any chance to put it on build.opensuse.org? It seems to be working well with APT

Is Mercury still being updated?

**Reply by rafalkrupinski** [reply]
Doesn't look like, but my previous comment was from two weeks after the last commit.

BTW, I've been using LibreWolf, but it breaks too many websites, so I moved to Zen.
